app/src/main/java/com/github/sdpteam15/polyevents/util/ItemTypeAdapter.kt

```diff
@@ -0,0 +1,12 @@
+package com.github.sdpteam15.polyevents.util
+
+import com.github.sdpteam15.polyevents.database.DatabaseConstant.ItemConstants.ITEM_TYPE
+
+object ItemTypeAdapter : AdapterInterface<String> {
+    override fun toDocument(element: String): HashMap<String, Any?> =
+        hashMapOf(ITEM_TYPE.value to element)
+
+    override fun fromDocument(document: MutableMap<String, Any?>, id: String): String =
+        document[ITEM_TYPE.value] as String
+
+}
```

Comment:
Do we still need this adapter?  ItemEntityAdapter can take care of converting Item types no?

Reply:
The goal is to be able to use the adapter with the generic methods made by Robin in FirestoreDatabaseProvider.  They need an Adapter for each different object stored online. So we can not use ItemEntityAdapter there.